src/components/common/ui/form/numbered-text-area-mui/NumberedTextArea.tsx

```diff
@@ -49,21 +38,33 @@ const NumberedTextArea: React.FC<NumberedTextAreaProps> = ({
     }
   };
 
+  const handleChange = (event: ChangeEvent<HTMLInputElement>) => {
+    const formattedValue = transformValue(event.target.value);
+    setInput(formattedValue);
+    if (formattedValue.trim() === '') {
+      setTouched(true);
+    } else {
+      setTouched(false);
```

Comment:
Why do we need to set touched false? Once the field is touched, it's touched (i think)